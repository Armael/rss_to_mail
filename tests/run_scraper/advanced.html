<html>
  <head>
    <title>T</title>
  </head>
  <body>
    <div class="article">
      <h2><a href="l1">#</a> Title 1</h2>
      <p>Blabla.</p>
      <p>Content 1</p>
    </div>
    <div class="article">
      <h2><a href="l2">#</a> Title 2</h2>
      <p>Blibli.</p>
      <p>Content 2</p>
    </div>
  </body>
</html>
