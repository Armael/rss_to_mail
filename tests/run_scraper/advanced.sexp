(".article"
 (T
  (entry
   (R
    ("> p" (T summary))
    ("> h2" (T title))
    ("> h2 > a" (T link))))))
("title" (T feed_title))
