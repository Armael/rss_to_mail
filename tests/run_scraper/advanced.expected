Feed title: None
Feed link: None
Feed icon: None
Entries (2):
	id: None
	title: # Title 2
	link: l2
	summary: Content 2
	id: None
	title: # Title 1
	link: l1
	summary: Content 1
